package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-logic-switch",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Do we want to bump the version?

Reply:
hmmm no idea, we can discuss it offline 😄

Reply:
Version number will now be updated only when a PR is manually merged.